package.json

```diff
@@ -35,6 +35,7 @@
     "chai": "^4.1.0",
     "eslint": "^4.3.0",
     "eslint-config-shellscape": "^1.0.0",
+    "express": "^4.15.4",
```

Comment:
Out of the box, Koa can't listen on https currently, making the alternative heavier than express. https://github.com/koajs/koa/issues/960
Luckily it's just a devDependency.